features/step_definitions/api-token.js

```diff
@@ -68,4 +68,51 @@ module.exports = function server() {
 
             Assert.notEqual(lastUsed, '');
         }));
+
+    this.Given(/^owns an existing API token named "([^"]*)"$/, tokenName =>
+        request({
+            uri: `${this.instance}/${this.namespace}/tokens`,
+            method: 'POST',
+            auth: {
+                bearer: this.jwt
+            },
+            body: {
+                name: tokenName
+            },
+            json: true
+        }).then((response) => {
+            Assert.oneOf(response.statusCode, [409, 201]);
+        }));
+
+    this.When(/^"calvin" lists all their tokens$/, () =>
```

Comment:
We should do a regex for the user's name, similar to how you did for the tokenName above